test/unit/spec/MfaVerify_spec.js

```diff
@@ -1110,6 +1110,36 @@ function (Okta,
             });
           });
       });
+      it('on WARNING_TIMEOUT, after send/re-send, shows warning message for SMS after 30s idle', function () {
```

Comment:
same here, text should be just `shows warning message to click "Re-send" after 30s`

Reply:
fixed